src/components/atoms/Cursor.tsx

```diff
@@ -10,11 +11,22 @@ const Cursor: React.FC = () => {
   const offsetWidth = textRef.current?.offsetWidth ?? 0
   const centeredTextValue = `calc(${movingCursor}px - ${offsetWidth / 2}px)`;
 
+  var IsFeature_enabled = feature == '123V1sdsd!';
```

Comment:
```suggestion
  var IsFeature_enabled = feature == '123V1sdsd!';
  const isFeatureEnabled = feature === XXX
```

Where does this magic number come from?